### DISCUSSION TITLE
Where can I find the logs?

### DISCUSSION THREAD

**Discussion by jose1711** [borgbase/vorta#984] [FAQ] [answered]
`vorta` gave me a message that there were some issues during the backup and I should check logs for details but I was unable to find them. Where are they? Can I also find the logs from past runs?

**Comment by m3nu** [comment, marked as answer]
Misc tab > Logs

**Reply by jose1711** [reply]
> Misc tab > Logs

Thank you. Hm... that's very well hidden :-) I was hoping for a dedicated view with filter (type of issues), color-coding of warnings, errors, etc. Hopefully one day someone improves this. For the time being though perhaps a hint for a user (like you did now) would be enough.

**Reply by LeoniePhiline** [reply]
I cannot open these logs. How can I see verbose borg output for each run?

**Reply by goebbe** [reply]
@LeoniePhiline In Vorta 9.1. the logs can now be found in the About-tab.  

In the left pane, at the bottom, select "Setting/  About", go to the About-tab: There should be a link to the logs.

**Reply by LeoniePhiline** [reply]
@goebbe Thank you! 

It seems like passing [`--list`](https://borgbackup.readthedocs.io/en/stable/usage/create.html) to `borg create` does not make verbose borg output appear in the logs, though. Can I do anything about this?

![image](https://github.com/borgbase/vorta/assets/22329650/878866a1-4acd-4224-8b68-6fd269b9c722)

**Reply by goebbe** [reply]
There is an issue in borg for showing the `--filter=AM` status when using `--dry-run`: 
https://github.com/borgbackup/borg/issues/3624

Currently borg only shows added (A) or modified (M) files if `borg create` actually adds or modifies files/paths. 
If you remove the `--dry-run` option from your borg arguments, the added and modified files should appear in the logs.

**Reply by LeoniePhiline** [reply]
Thanks! Such a pity, I used dry run to try and verify my exclusion globs before starting a real upload :D 

Thank you very much for your responses!

**Reply by LeoniePhiline** [reply]
> There is an issue in borg for showing the `--filter=AM` status when using `--dry-run`: 
> https://github.com/borgbackup/borg/issues/3624
> 
> Currently borg only shows added (A) or modified (M) files if `borg create` actually adds or modifies files/paths. 
> If you remove the `--dry-run` option from your borg arguments, the added and modified files should appear in the logs.
> 

I cannot find file level logging with `--list` despite running without dry run. Is this expected behavior?

**Reply by goebbe** [reply]
I just tested this on my computer: Using Vorta, the output of  `borg create --list --filter=AM ` is not in the logs.  
I don't know enough about the internals of Vorta to tell if this is the intended behavior. 
If I run borg commands with `--list` directly, on the command line, the expected output gets printed.  
Why adding `--list` and `--filter` by default - if the corresponding output is not printed to the logs? 
@LeoniePhiline Maybe you could open an issue to clarify this?

**Reply by LeoniePhiline** [reply]
I've created https://github.com/borgbase/vorta/issues/1982.

**Comment by ivystopia** [comment]
Logs location changed in v0.9.1 and newer.

Settings/About > About > "Click here to view the logs"
